src/chrome/content/rules/electroneum.com.xml

```diff
@@ -0,0 +1,14 @@
+
+<ruleset name="electroneum.com">
+
+	<target host="electroneum.com" />
+	<target host="*.electroneum.com" />
```

Comment:
Is there a specific reason for using a wildcard?

Reply:
Because they have a wildcard certificate. If you want i can replace the wildcard.

Reply:
We try to avoid wildcards, because they are harder to maintain.